internal/config/config.go

```diff
@@ -0,0 +1,121 @@
+package config
+
+import (
+	"fmt"
+	"github.com/rs/zerolog"
+	"strings"
+	"time"
+
+	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
+	"github.com/spf13/viper"
+)
+
+type DatabaseT struct {
+	Host        string
+	Port        uint16
+	Name        string
+	User        string
+	Password    string
+	MaxIdleTime time.Duration
+	MaxLifetime time.Duration
+	MaxOpenConn int
+	MaxIdleConn int
+	LogLevel    int
+}
+
+type LoggingT struct {
+	Level  int
+	Stdout bool
+}
+type CloudwatchT struct {
+	Enabled bool
+	Group   string
+	Stream  string
+}
+type FeatureFlagsT struct {
+	Environment string
+	ExitOnPanic bool
+}
+
+type GlobalConfigT struct {
+	Database DatabaseT
+	Logging LoggingT
+	Cloudwatch CloudwatchT
+	Features FeatureFlagsT
+}
+
+var config GlobalConfigT
+var Database *DatabaseT
+var Logging *LoggingT
+var Cloudwatch *CloudwatchT
+var Features *FeatureFlagsT
```

Comment:
Uh those T suffixes are ugly, why you have them? It's completely okay to have the very same variable and type name, it's actually a very good design pattern to follow.

Also can you drop `GlobalConfigT` type and `config` variable completely? They appear to be unused, at least I don't see any use.

Reply:
alright, I've figured how to do that, it's working now :tada: